go/config/config_test.go

```diff
@@ -47,4 +47,19 @@ var _ = Describe("ElasticsearchConfig", func() {
 			Refresh: "somethingInvalid",
 		}, true),
 	)
+
+	When("setting the InsecureSkipVerify boolean value", func() {
+		It("shoulbe be true when set to true", func() {
+			abc := &ElasticsearchConfig{
+				InsecureSkipVerify: true,
+			}
+			Expect(abc.InsecureSkipVerify).Should(BeTrue())
+		})
+		It("shoulbe be false when set to false", func() {
```

Comment:
can you fix the spelling here and on line 52?

Reply:
sure. was this the correct approach for testing this? we don't really have validation on this field